test/loader/serializeInstructionTest.cpp

```diff
@@ -1126,4 +1242,360 @@ TEST(SerializeInstructionTest, SerializeDotProductInstruction) {
   Output = {};
   EXPECT_FALSE(Ser.serializeSection(createCodeSec(Instructions), Output));
 }
+
+TEST(SerializeInstructionTest, SerializeConvertInstruction) {
```

Comment:
These instructions are also reference instrucitons. Combine them into the test set.